defixit/anonlytics-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "bundles": {
+    "DeFixIT\\AnonlyticsBundle\\AnonlyticsBundle": ["all"]
+  },
+  "copy-from-recipe": {
+    "config/": "%CONFIG_DIR%/"
+  },
+  "env": {
+    "CLIENT_TOKEN": "",
+    "SITE_TOKEN": ""
```

Comment:
Could you please add a prefix to these envars? `ANONLYTICS_SITE_TOKEN`?

Reply:
Yes will do that now